Meadowlark-js/yarn.lock

```diff
@@ -12240,7 +12240,9 @@ __metadata:
     eslint-plugin-prettier: 4.0.0
     husky: ^7.0.4
     jest: 27.5.1
+    jest-environment-node: ^27.0.0
     lerna: ^3.22.1
+    mongodb: ^4.5.0
```

Comment:
Fixes unrelated warnings on `yarn install` from prior work.

Reply:
Shouldn't this be fixed to 4.0.x?

Reply:
Keep in mind this is the client driver version and not the MongoDB version. They recommend using the latest driver version that supports the server you use: https://www.mongodb.com/docs/drivers/node/current/compatibility/